### PR TITLE
compleated all tasks and passed all tests

### PR DESCRIPTION
**Pull request by TKuts**
compleated all tasks and passed all tests

### CODE REVIEW

File: src/transformState.js [hunk 1]
```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(howTransform => {
```
Comment (polosanya) [inline src/transformState.js]:
```suggestion
  actions.forEach(action => {
```
image

File: src/transformState.js [hunk 2]
```diff
@@ -5,7 +5,27 @@
...
+          delete state[key];
+        }
+        break;
+      case 'removeProperties':
+        keysToRemove.forEach(propKey => {
+          if (state.hasOwnProperty(propKey)) {
+            delete state[propKey];
+          }
+        });
+        break;
```
Comment (polosanya) [inline src/transformState.js:26]: image

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost done!

**Review by demetrxx — APPROVED** [review]
Well done 👍 Go ahead!

**Pull request closed without merging**